modules/prelude-dart.el

```diff
@@ -0,0 +1,58 @@
+;;; prelude-dart.el --- Emacs Prelude: Clojure programming configuration.
```

Comment:
This mentions "Clojure"; should this be changed to "Dart"?

Reply:
I guess so. :-)

Reply:
Got it, done! 🙃